build/ci/github_actions/operator/deploy_driver.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -xel
+set +o pipefail
+
+install_worker_prerequisites() {
+  kind_node_name=`docker ps --format "{{.Names}}"`
+  docker exec -i $kind_node_name apt-get update
+  docker exec -i $kind_node_name apt -y install open-iscsi
+}
+
+edit_cr_images (){
+  cd $(dirname $cr_file)
+  chmod 547 $(basename $cr_file)
+  declare -A cr_image_fields=(
+      [".spec.controller.repository"]="$controller_repository_for_test"
+      [".spec.controller.tag"]="$driver_images_tag"
+      [".spec.node.repository"]="$node_repository_for_test"
+      [".spec.node.tag"]="$driver_images_tag"
+  )
+  for image_field in ${!cr_image_fields[@]}; do
+      cr_image_value=${cr_image_fields[${image_field}]}
+      yq eval "${image_field} |= \"${cr_image_value}\"" $(basename $cr_file) -i
+  done
+  cd -
+}
+
+edit_operator_yaml_image (){
```

Comment:
would it still be called `operator.yaml` after the [sdk version bump](https://github.com/IBM/ibm-block-csi-operator/pull/195)?